src/entities.ts

```diff
@@ -39,53 +42,53 @@ export const newL2BridgeZapInstance = (params: {
-        return L1BridgeZapFactory.connect(address, params.signerOrProvider);
-    }
+        bridgeConfigAddress:   string = "0x7fd806049608b7d04076b8187dd773343e0589e6",
+        bridgeConfigV3Address: string = "0x3ee02f08B801B1990AC844d8CD2F119BA6Fb9bcF",
+        poolConfigAddress:     string = "0xB34C67DB5F0Fd8D3D4238FD0A1cBbfD50a72e177";
 
-    export function l2BridgeZap(params: {
-        chainId: number,
-        signerOrProvider?: SignerOrProvider
-    }): L2BridgeZapContract {
-        const address: string = contractAddressFor(params.chainId, "bridge_zap");
-        return L2BridgeZapFactory.connect(address, params.signerOrProvider);
+    export interface NewEntityParams {
+        chainId:           number;
+        signerOrProvider?: SignerOrProvider;
     }
 
-    export function zapBridge(params: {
-        chainId: number,
-        signerOrProvider?: SignerOrProvider
-    }): GenericZapBridgeContract {
-        const address: string = contractAddressFor(params.chainId, "bridge_zap");
-
-        if (params.chainId === ChainId.ETH) {
-            return L1BridgeZapFactory.connect(address, params.signerOrProvider)
-        }
-
-        return L2BridgeZapFactory.connect(address, params.signerOrProvider)
-    }
-
-    export function bridgeConfig(): BridgeConfigContract {
-        const provider = newProviderForNetwork(ChainId.ETH);
-        return BridgeConfigFactory.connect(bridgeConfigAddress, provider)
-    }
-
-    export function poolConfig(): PoolConfigContract {
-        const provider = newProviderForNetwork(ChainId.ETH);
-        return PoolConfigFactory.connect(poolConfigAddress, provider)
-    }
+    export const synapseBridge = (params: NewEntityParams): SynapseBridgeContract =>
+        newSynapseBridgeInstance({
+            address: contractAddressFor(params.chainId, "bridge"),
+            ...params
+        })
+
+    export const l1BridgeZap = (params: NewEntityParams): L1BridgeZapContract =>
+        newL1BridgeZapInstance({
+            address: contractAddressFor(params.chainId, "bridge_zap"),
+            ...params
+        })
+
+    export const l2BridgeZap = (params: NewEntityParams): L2BridgeZapContract =>
+        newL2BridgeZapInstance({
+            address: contractAddressFor(params.chainId, "bridge_zap"),
+            ...params
+        })
+
+    export const zapBridge = (params: NewEntityParams): GenericZapBridgeContract =>
+        params.chainId === ChainId.ETH
+            ? l1BridgeZap(params)
+            : l2BridgeZap(params)
+
+    export const bridgeConfig = (): BridgeConfigContract =>
+        BridgeConfigFactory.connect(
+            bridgeConfigAddress,
+            rpcProviderForNetwork(ChainId.ETH)
+        )
+
+    export const bridgeConfigV3 = (): BridgeConfigV3Contract =>
+        BridgeConfigV3Factory.connect(
+            bridgeConfigV3Address,
+            rpcProviderForNetwork(ChainId.ETH)
+        )
+
+    export const poolConfig = (): PoolConfigContract =>
```

Comment:
PoolConfig is now merged into BridgeConfigV3 which stores poolconfig: https://github.com/synapsecns/synapse-contracts/blob/162745e7c2e7a5d2012f1b12ea6bc95391ca4c3c/contracts/bridge/BridgeConfigV3.sol#L399

Reply:
Fixed in [e5fe607](https://github.com/synapsecns/sdk/pull/63/commits/e5fe60795630afe9c371ab3fb947aca069f8d7bc)